src/types.ts

```diff
@@ -40,3 +59,38 @@ export type BallotMetadata = Omit<
   BallotPageMetadata,
   'pageNumber' | 'pageCount'
 >
+
+export type SerializableBallotPageLayout = Omit<
+  BallotPageLayout,
+  'ballotImage'
+> & {
+  ballotImage: Omit<BallotPageLayout['ballotImage'], 'imageData'>
+}
+
+export * from './types/ballot-review'
+
+export function isMarked(
+  mark: BallotTargetMark,
+  { marginalMarkMin = 0.1, validMarkMin = 0.8 } = {}
+): boolean | undefined {
+  if (mark.score >= validMarkMin) {
+    return true
+  }
+
+  if (mark.score < marginalMarkMin) {
+    return false
+  }
+}
```

Comment:
implicit `undefined` return... had me confused for a bit.

Reply:
Done